File: Java/Os_class.java
public class Os_class
{
    private String namaOS;
    private String versi;

    public Os_class()
    {
        this.namaOS = "";
        this.versi = "";
    }

    public Os_class(String namaOS, String versi)
    {
        this.namaOS = namaOS;
        this.versi = versi;
    }

    public void set_namaOS(String namaOS)
    {
        this.namaOS = namaOS;
    }

    public void set_versi(String versi)
    {
        this.versi = versi;
    }

    public String get_namaOS()
    {
        return this.namaOS;
    }

    public String get_versi()
    {
        return this.versi;
    }

}

File: Python/Os_class.py
class OS:
    __namaOS = ""
    __versi = ""

    def __init__(self, namaOS, versi):
        self.__namaOS = namaOS
        self.__versi = versi

    def get_namaOS(self):
        return self.__namaOS
    
    def get_versi(self):
        return self.__versi
    
    def set_namaOS(self, namaOS):
        self.__namaOS = namaOS

    def set_versi(self, versi):
        self.__versi = versi